website/docs/r/app_service.html.markdown

```diff
@@ -168,6 +168,8 @@ The following attributes are exported:
 
 * `default_site_hostname` - The Default Hostname associated with the App Service - such as `mysite.azurewebsites.net`
 
+* `outbound_ip_addresses` - A comma separated list of outbound IP addresses - such as `52.23.25.3,52.143.43.12.5`
```

Comment:
_minor_ I think we want to remove the last segment from the second IPv4 address :)